ebos/eclproblem.hh

```diff
@@ -2091,7 +2081,7 @@ protected:
             if (!this->maxOilSaturation_.empty())
                 this->maxOilSaturation_[elemIdx] = std::max(this->maxOilSaturation_[elemIdx], fs.saturation(oilPhaseIdx));
             if (!this->minOilPressure_.empty())
-                this->minOilPressure_[elemIdx] = std::min(this->minOilPressure_[elemIdx], fs.pressure(oilPhaseIdx));
+                this->minOilPressure_[elemIdx] = std::min(this->minOilPressure_[elemIdx], fs.pressure(refPressurePhaseIdx_()));
```

Comment:
The other quantities here, $S_w^{\mathrm{max}}$ and $S_o^{\mathrm{max}}$, both use explicit phase indices.  Why is it that we only vary the phase index for the pressure quantity?

Reply:
Both maxWaterSaturation_ and maxOilSaturation_ are empty if the phase does not exist. Pressure is needed for all combinations of phases. I will rename minOilPressure_ to minRefPressure_ to avoid further confusion.